### PR TITLE
Slap My Hand

### PR DESCRIPTION
**Pull request by Cenrus**
https://github.com/yogstation13/Yogstation/pull/4405

## About this Pull Request

Ports High fives from YogStation 

## Why it's good for the game
52241965-383fdb00-28a3-11e9-9a6c-acb1beb9951c

## Changelog

:cl: Theos
rscadd: Adds high-fiving.
/:cl:

### CODE REVIEW

**Review by MarkSuckerberg — APPROVED** [review]
nice

**Comment by SomeAngryMiner** [comment]
extremely important feature.

**Comment by Cenrus** [comment]
I tested this and it works.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -694,3 +694,29 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	desc = "Slap my hand."
+	icon_state = "latexballon"
+	item_state = "nothing"
+	hitsound = 'sound/weapons/punchmiss.ogg'
+	force = 0
+	throwforce = 0
+	item_flags = DROPDEL | ABSTRACT
+	attack_verb = list("is left hanging by")
+
+obj/item/highfive/attack(mob/target, mob/user)
```
Comment (ike709) [inline code/game/objects/items/weaponry.dm]:
```suggestion
/obj/item/highfive/attack(mob/target, mob/user)
```

**Comment by ike709** [comment]
This doesn't seem to indicate to anybody else that a user is trying to high five. Should add a visible message to run_emote() or something.

**Comment by ike709** [comment]
Tested?

**Comment by Cenrus** [comment]
Yes, there is a third person message now.

**Pull request merged**